testnets/_IBC/outbedevnet-cosmoshubtestnet.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "outbedevnet",
+    "client_id": "07-tendermint-3",
+    "connection_id": "connection-5"
+  },
+  "chain_2": {
+    "chain_name": "provider",
```

Comment:
'provider' is not a valid chain_name. Do you mean cosmosicsprovidertestnet?

Reply:
the same chain_name must also be specified in the file name (and be in alphabetical error)